library/books-api-v2/index.ts

```diff
@@ -0,0 +1,48 @@
+import express from "express";
+import { Book } from "./v2/Book";
+import { BookMapper } from "./BookMapper"
+
+const PORT = parseInt(process.env.PORT ?? "8080");
+
+const app = express();
+
+app.get("/ping", (_req, res) => {
+  res.send("pong");
+});
+
+app.get("/api/v1/books", (_req, res) => {
+  res.send(getBooks().map(BookMapper.toV1));
```

Comment:
Following my previous suggestion, this just becomes:
```suggestion
  res.send(getBooks().map(bookToV1));
```
Assuming you import it just as
```ts
import { bookToV1 } from './BookMapper';
```